### ISSUE TITLE
[Adaptive Card : Labels and instructions]: Screen reader does not announces the name of place when focus lands on 'James Russel' card. 

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7239]
https://microsoft.visualstudio.com/OS/_workitems/edit/33201369

**Comment by vagpt** [comment]
Closing this issue as of now this issue is being tracked with below URL.

https://github.com/microsoft/AdaptiveCards/issues/7713
